core/client/client_scion.go

```diff
@@ -152,6 +156,17 @@ func (c *SCIONClient) measureClockOffsetSCION(ctx context.Context, log *zap.Logg
 	srcAddr := &net.IPAddr{IP: localAddr.Host.IP}
 	dstAddr := &net.IPAddr{IP: remoteAddr.Host.IP}
 
+	var ntskeData ntske.Data
```

Comment:
This code block should land just below of the line
```
localPort := conn.LocalAddr().(*net.UDPAddr).Port
```
Otherwise, `remoteAddr` will have the wrong value.